.env.example

```
@@ -3,14 +3,47 @@ POSTGRES_USER=
+RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS=
+RABBITMQ_EXCHANGE=
+RABBITMQ_NOTIFICATIONS_QUEUE=
+RABBITMQ_DOCUMENT_CLEANUP_QUEUE=
+RABBITMQ_PREFETCH=
+RABBITMQ_CONSUMER_RETRY_INTERVAL_MS=
+ASYNC_OUTBOX_PUBLISHER_ENABLED=
+ASYNC_OUTBOX_PUBLISHER_INTERVAL_MS=
+ASYNC_OUTBOX_PUBLISHER_BATCH_SIZE=
+ASYNC_OUTBOX_DATABASE_RETRY_ATTEMPTS=
+ASYNC_OUTBOX_DATABASE_RETRY_BASE_DELAY_MS=
+DOCUMENT_CLEANUP_INLINE_PROCESSING_ENABLED=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DOCUMENT_CLEANUP_INLINE_PROCESSING_ENABLED key should go before the RABBITMQ_CONSUMER_RETRY_INTERVAL_MS key

(UnorderedKey)